### ISSUE TITLE
No memory temps for RTX 3080 with 0.25.9

### ISSUE THREAD

**Issue by kkoouu** [trexminer/T-Rex#1169]
Hi,

just tested new release, tried to run t-rex under root but I can't see memory temps.

According to specs my RTX 3080 SUPRIM X 10G has GDDR6X.

ps output:

```
kou@woody:~$ ps -ef |grep t-rex
root     10722 10721  0 12:54 pts/5    00:00:00 ./t-rex -a ethash -o stratum+tcp://eth.2miners.com:2020 -u 0x07<cut>348 -p x -w woody --api-bind-http 0.0.0.0:4067 -l /home/kou/scripts/trex/trex.log --gpu-report-interval 30
root     10734 10722 17 12:54 pts/5    00:00:10 /home/kou/scripts/trex/t-rex --watchdog-version 0.25.9 -a ethash -o stratum+tcp://eth.2miners.com:2020 -u 0x07<cut>348 -p x -w woody --api-bind-http 0.0.0.0:4067 -l /home/kou/scripts/trex/trex.log --gpu-report-interval 30 --watchdog-pipe /tmp/trex10722
```

log:
```
20220317 12:54:25 T-Rex NVIDIA GPU miner v0.25.9  -  [Linux]
20220317 12:54:25 r.fcd1d0561127
20220317 12:54:25 
20220317 12:54:25 
20220317 12:54:25 NVIDIA Driver v470.103.01
20220317 12:54:25 
20220317 12:54:25 + GPU #0: [00:02.0|2206] GeForce RTX 3080, 10018 MB
20220317 12:54:25 + GPU #1: [00:04.0|2206] GeForce RTX 3080, 10018 MB
20220317 12:54:25 
20220317 12:54:25 WARN: DevFee 1% (ethash)
20220317 12:54:25 
20220317 12:54:25 URL : stratum+tcp://eth.2miners.com:2020
20220317 12:54:25 USER: 0x07<cut>348
20220317 12:54:25 PASS: x
20220317 12:54:25 WRK : woody
20220317 12:54:25 
20220317 12:54:25 Starting on: eth.2miners.com:2020
20220317 12:54:25 WARN: API server is not bound to 127.0.0.1, consider setting up API key (see --api-key)
20220317 12:54:25 ApiServer: HTTP server started on 0.0.0.0:4067
20220317 12:54:25 --------------------------------------------------------
20220317 12:54:25 For control navigate to: http://192.168.13.140:4067/trex
20220317 12:54:25 --------------------------------------------------------
20220317 12:54:25 Using protocol: stratum1.
20220317 12:54:25 Authorizing...
20220317 12:54:25 Authorized successfully.
20220317 12:54:25 ethash epoch: 480, block: 14403870, diff: 8.73 G
20220317 12:54:25 GPU #0: intensity 22
20220317 12:54:25 GPU #1: intensity 22
20220317 12:54:28 GPU #1: generating DAG 4.75 GB for epoch 480 ...
20220317 12:54:28 GPU #0: generating DAG 4.75 GB for epoch 480 ...
20220317 12:54:37 GPU #1: DAG generated [crc: 415ab8eb, time: 8725 ms], memory left: 4.80 GB
20220317 12:54:37 GPU #0: DAG generated [crc: 415ab8eb, time: 8762 ms], memory left: 4.80 GB
20220317 12:54:40 ethash epoch: 480, block: 14403871, diff: 8.73 G
20220317 12:54:52 GPU #1: using kernel #4
20220317 12:54:52 GPU #0: using kernel #4
20220317 12:55:05 ethash epoch: 480, block: 14403872, diff: 8.73 G

--------------------20220317 12:55:25 ---------------------
Mining at eth.2miners.com:2020 [51.195.4.174], diff: 8.73 G
GPU #0: RTX 3080 - 95.15 MH/s, [T:34C, P:209W, F:70%, E:498kH/W]
GPU #1: RTX 3080 - 98.68 MH/s, [T:36C, P:215W, F:70%, E:506kH/W]
Hashrate: 193.83 MH/s, Shares/min: 0, Avg.P: 386W, Avg.E: 502kH/W
Uptime: 1 min | Algo: ethash | T-Rex v0.25.9
```

**Comment by trexminer** [comment]
Ok, thanks.
What's your linux distro? My guess is the kernel is sealed.

**Comment by kkoouu** [comment]
Standard Debian Buster:
kou@woody:~$ cat /etc/debian_version 
10.11
kou@woody:~$ uname -a
Linux woody 4.19.0-19-amd64 #1 SMP Debian 4.19.232-1 (2022-03-07) x86_64 GNU/Linux

**Comment by trexminer** [comment]
I may have found a way to fix it and also make it work without root, give this one a go please: https://trex-miner.com/download/test/t-rex-0.25.9-debug2-linux.tar.gz

**Comment by kkoouu** [comment]
OK, it's working fine now even without root, thanks !

**Comment by jiiimjooon** [comment]
Mine is now working too, thanx :^)

**Comment by jemshit** [comment]
@trexminer you creating a release anytime soon?

**Comment by sevagh** [comment]
debug2 working for me on my Ampere cards on Fedora 35 with kernel 5.6.11

**Comment by trexminer** [comment]
fixed in 0.25.12
